app/app_notlegacy_darwin.go

```diff
@@ -0,0 +1,9 @@
+// +build !ci
+// +build !legacy
```

Comment:
Is the code dedicated in the current release branch and will never be contained in future releases? If we have everything in the develop compatible with go:build whereas this newly introduced one is not, then when the develop branch is merged into the release branch, we will have missed cases such as this one.

Having an additional go:build should be no problem at all since they are useless for those older compilers. If 1.18 is released, then users who use the newest compiler will not be able to use the current release as a module because their compiler cannot recognize the build tags. So in any case, we will have to adapt the go:build tag for every supported release.

Reply:
> Is the code dedicated in the current release branch and will never be contained in future releases?

When the release branch does merge to develop this will be addressed - the linters will not allow it in without that code added :). However adding it to this PR and the branch would be in an inconsistent state.
I don't think we will still be working on v2.0.x bug fix releases when Go 1.18 is released. If we are then this branch can be updated.